### PR TITLE
[ST-992] Improve wovnjava docker environment

### PR DESCRIPTION
**Pull request by Soichiro725**
https://wovnio.atlassian.net/browse/ST-992

This PR make it easier to use local docker environment.

- Add make commands to build and update website easier
- Using local wovnjava as default instead of using JitPack version.
  I made pox.xml and pom_jitpack.xml, so still easy to use JitPack version.
- Add knowledge to docker/README.md
- Mount logs files to see logs inside docker easily
- Skip test by adding `-Dmaven.test.skip` option to make compile step fast

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,21 +2,43 @@ MAKEFLAGS    += --warn-undefined-variables
...
 test:
 	$(MAVEN) test -f pom.jdk$(VERSION).xml
+
+start:
+	docker-compose -f docker/java$(VERSION)/docker-compose.yml up
+
+stop:
+	docker-compose -f docker/java$(VERSION)/docker-compose.yml rm -sf
+
+build_webisite:
```
Comment (bstewart00) [inline Makefile:29]: `build_website`

File: docker/README.md [hunk 2]
```diff
@@ -1,56 +1,109 @@
-# Example of Docker
+# Release
+## Release steps
+1. Update `version` in pom.xml on root directory.
+2. Make release in GitHub.
+3. JitPack builds wovnjava library. (https://jitpack.io/#wovnio/wovnjava)  
+
+## JitPack
+Our wovnjava repogitory is registered in JitPack, and JitPack is monitoring our repogitory.
```
Comment (bstewart00) [inline docker/README.md:8]: `repository`

File: docker/README.md [hunk 3]
```diff
@@ -1,56 +1,109 @@
...
 ```
+This command start tomcat to serve your website.  
+Go to http://localhost:8080 , then you can see your website.
 
-## 6. Restart Tomcat
+## Change your website
+When you want to change your website, change `docker/java8/hello/src/main/webapp`.  
+The following command rebuilds your website, and restart tomcat to apply them.
+```
+make build_webisite && make restart
```
Comment (bstewart00) [inline docker/README.md:78]: `build_website`

File: Makefile [hunk 4]
```diff
@@ -2,21 +2,43 @@ MAKEFLAGS    += --warn-undefined-variables
...
-
 VERSION := 8
-MAVEN    = docker run -it --rm -v ${PWD}:/project -v wovnjava-maven_repo:/root/.m2 -w /project maven:3-jdk-$(VERSION) mvn
+TARGET_DIR = ${PWD}
+MAVEN    = docker run -it --rm -v ${TARGET_DIR}:/project -v wovnjava-maven_repo:/root/.m2 -w /project maven:3-jdk-$(VERSION) mvn
+WEBSITE_CONFIG_FILE = pom.xml
+# WEBSITE_CONFIG_FILE = pom_jitpack.xml
 
-.PHONY: all
+.PHONY: all clean build test start stop build_webisite build_wovn_java build_webisite_with_loacal_wovn_java restart
```
Comment (bstewart00) [inline Makefile:11]: `build_website_with_local_wovnjava`

**Review by bstewart00 — COMMENTED** [review]
These commands make it much easier to use 👍 . I can make local changes to wovnjava and run `make build_wovn_java_and_website` and be confident that the changes are applied. The log file mapping is also good.

**Review by bstewart00 — APPROVED** [review]

**Review by zeyuwu — APPROVED** [review]
Looks good.

**Review by iwai-wovn — APPROVED** [review]

**Pull request merged**